<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <arg name="resolution"  default="0.1"/>
    <arg name="map_resolution"  default="0.6"/>
    
    <node pkg="people_lib_study" name="detect_item" type="detect_item" output="screen">
        <param name="compare_pcd_file"  value="/mnt/uclab.pcd"/>
        <param name="resolution" value="0.15" />
        <param name="map_resolution" value="0.6" />
        <param name="maxSize" value="1"/> 
        <param name="input_topic" value="/points_no_ground" />
        <param name="output_topic" value="/points_no_wall" />
        <param name="finishRemovePlaneRatio" value="0.5" />
        <param name="removeThreshold" value="0.1" />
        <param name="angular_threshold" value="0.1" />
    </node>
    <!-- <node pkg="people_lib_study" name="detect_item" type="detect_item" output="screen">
        <param name="compare_pcd_file"  value="/mnt/dicomo1.pcd"/>
        <param name="resolution" value="0.1" />
        <param name="map_resolution" value="1.0" />
        <param name="maxSize" value="1"/> 
        <param name="input_topic" value="/points_no_ground" />
        <param name="output_topic" value="/points_no_wall" />
        <param name="finishRemovePlaneRatio" value="0.5" />
        <param name="removeThreshold" value="0.2" />
    </node> -->
</launch>